multicast/__main__.py

```
@@ -626,18 +626,18 @@ def doStep(self, *args, **kwargs):
 				__EXIT_MSG = (70, _TOOL_MSG)
 				if (sys.stdout.isatty()):  # pragma: no cover
 					print(str(_TOOL_MSG))
-		except Exception as err:  # pragma: no branch
-			exit_code = exceptions.get_exit_code_from_exception(err)
+		except Exception as _cause:  # pragma: no branch
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)